plugins/org.eclipse.fordiac.ide.systemmanagement/src/org/eclipse/fordiac/ide/systemmanagement/changelistener/FordiacResourceChangeListener.java

```diff
@@ -192,8 +194,23 @@ private static boolean handleResourceMovedFrom(final IResourceDelta delta) {
 		return false;
 	}
 
+	private static void validateProjectNature(final IProject project) {
+		try {
+			if (project.getNature(SystemManager.FORDIAC_PROJECT_NATURE_ID) instanceof final FordiacNature nature) {
+				nature.validate();
+			}
+		} catch (final CoreException e) {
+			FordiacLogHelper.logError(e.getMessage());
```

Comment:
please use `logError(e.getMessage(), e)`; so that we get better info in the stack trace. Maybe even add a dedicated message instead of `e.getMessage()`.